giskard-frontend/src/views/main/project/PredictionResults.vue

```diff
@@ -41,7 +41,26 @@
             <div class="caption">
               <div v-if="targetFeature">target: {{ targetFeature }}</div>
               <div v-if="model && model.threshold">threshold: {{ model.threshold }}</div>
-              <div v-if="actual">{{ labelsAndValues }}</div>
+              <div id='labels-container' v-if='actual && isClassification(predictionTask)'>
+                <v-simple-table dense height="200px" fixed-header>
+                  <template v-slot:default>
+                    <thead>
+                    <tr>
+                      <th scope='col' class="text-center">
+                        Labels
+                      </th>
+                    </tr></thead>
+                    <tbody>
+                    <tr
+                      v-for="label in classificationLabels.values()"
+                      :key="label"
+                    >
+                      <td>{{ label }}</td>
```

Comment:
can you also add indices of labels to the table? They may be useful for debugging the model